packages/lib/services/webhook.ts

```diff
@@ -83,11 +105,15 @@ export const updateWebhook = async (
 
 export const deleteWebhook = async (id: string): Promise<TWebhook> => {
   try {
-    return await prisma.webhook.delete({
+    let deletedWebhook = await prisma.webhook.delete({
```

Comment:
same here. if we have the right prisma types we can also skip these checks again and simplify the method again